templates/docs/examples/patterns/notifications/variants.html

```diff
@@ -11,112 +11,139 @@
 <h4>Borderless</h4>
 <div class="p-notification--caution is-borderless">
   <div class="p-notification__content">
-    <div class="p-notification__title">Title</div>
-    <div class="p-notification__message">Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.</div>
+    <h4 class="p-notification__title">Title</h4>
+    <p class="p-notification__message">
+      Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.
+    </p>
   </div>
 </div>
 
 <h4>Borderless inline</h4>
 <div class="p-notification--caution is-borderless is-inline">
   <div class="p-notification__content">
-    <div class="p-notification__title">Title:</div>
-    <div class="p-notification__message">Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.</div>
+    <h4 class="p-notification__title">Title:</h4>
+    <p class="p-notification__message">
+      Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.
+    </p>
   </div>
 </div>
 
 <h4>Collapsed (not implemented)</h4>
```

Comment:
I guess we should drop the stacked/collapsed example completely for now. We don't want to implement it as we are not sure in what context it will be used and how it should behave.